src/Razor/src/Microsoft.VisualStudio.RazorExtension/Microsoft.VisualStudio.RazorExtension.csproj

```diff
@@ -26,6 +26,9 @@
 
     <SetupProductArch>neutral</SetupProductArch>
     <ServiceHubCoreSubPath>ServiceHubCore</ServiceHubCoreSubPath>
+
+    <!-- for the syntax visualizer -->
+    <UseWPF>true</UseWPF>
```

Comment:
lmao, had no clue. What's this flag even do to the installation contents? Add a pkgdef thing or something?

Reply:
No, just adds references to the WPF assemblies.